commit 003000d451833309c963054e58a48fa1df7e767b
Author: Steve Dickson <steved@redhat.com>
Date:   Thu Dec 10 13:13:03 2020 -0500

    exportfs: Ingnore export failures in nfs-server.serivce unit
    
    With some recent commits, exportfs will continue on trying to
    export filesystems even when an entry is invalid or does
    not exist, but will still have a non-zero exit to report
    the error.
    
    This situation should not stop the nfs-server service
    from comingup so nfs-server.service file should
    ignore these types of failures
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/systemd/nfs-server.service b/systemd/nfs-server.service
index 06c1adb7..b432f910 100644
--- a/systemd/nfs-server.service
+++ b/systemd/nfs-server.service
@@ -21,13 +21,13 @@ After=rpc-gssd.service gssproxy.service rpc-svcgssd.service
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStartPre=/usr/sbin/exportfs -r
+ExecStartPre=-/usr/sbin/exportfs -r
 ExecStart=/usr/sbin/rpc.nfsd
 ExecStop=/usr/sbin/rpc.nfsd 0
 ExecStopPost=/usr/sbin/exportfs -au
 ExecStopPost=/usr/sbin/exportfs -f
 
-ExecReload=/usr/sbin/exportfs -r
+ExecReload=-/usr/sbin/exportfs -r
 
 [Install]
 WantedBy=multi-user.target
